cfgov/unprocessed/js/organisms/MegaMenuMobile.js

```diff
@@ -184,14 +244,35 @@ function MegaMenuMobile( menus ) {
    */
   function suspend() {
     if ( !_suspended ) {
-      _menus.getRoot().data.getTransition().remove();
+      _suspended = true;
 
-      // TODO: Update this to close the menus directly
+      _rootMenu.getTransition().remove();
+
+      var rootNode = _menus.getRoot();
+      treeTraversal.bfs( rootNode, function( node ) {
+        node.data.setCollapseTransition( null );
+        node.data.setExpandTransition( null );
+
+        // TODO: Investigate whether deferred collapse has another solution.
+        //       This check is necessary since a call to an already collapsed
+        //       menu will set a deferred collapse that will be called
+        //       on expandEnd next time the flyout is expanded.
+        //       The deferred collapse is used in cases where the
+        //       user clicks the flyout menu while it is animating open,
+        //       so that it appears like they can collapse it, even when
+        //       clicking during the expand animation.
+        if ( node.data.isExpanded() ) {
+          node.data.collapse();
+        }
+      } );
+
+      _rootMenu.getDom().content.classList.remove( 'u-invisible' );
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/1617/commits/ba618d387db8265534298e94c02f606c05b0def5